core/domain/email_manager_test.py

```diff
@@ -88,6 +90,198 @@ def test_sender_id_validation(self):
         # pylint: enable=protected-access
 
 
+class ExplorationMemebershipEmailTests(test_utils.GenericTestBase):
+    """Tests that sending exploration membership email works as expected."""
+
+    def setUp(self):
+        super(ExplorationMemebershipEmailTests, self).setUp()
+
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+
+        self.signup(self.NEW_USER_EMAIL, self.NEW_USER_USERNAME)
+        self.new_user_id = self.get_user_id_from_email(self.NEW_USER_EMAIL)
+
+        self.exploration = self.save_new_default_exploration(
+            'A', self.editor_id, 'Title')
+
+        self.email_subject_tempate = (
```

Comment:
Yes, this looks good. I suggest laying it out like this for easier reading (i.e. break lines after 'br'). I also made some textual and formatting amendments.

```
        expected_email_html_body = (
            'Hi newuser,<br>'
            '<br>'
            '<b>editor</b> has granted you manager rights to their learning exploration, '
            '<a href="http://www.oppia.org/A">"Title"</a>, on Oppia.org.<br>'
            '<br>'
            'This allows you to:<br>'
            '<ul>'
            '  <li>Change the exploration permissions</li>'
            '  <li>Edit the exploration</li>'
            '  <li>View and playtest the exploration</li>'
            '</ul><br>'
            'You can find the exploration <a href="http://www.oppia.org/A">here</a>.<br>'
            '<br>'
            'Thanks, and happy collaborating!<br>'
            '<br>'
            'Best Wishes,<br>'
            'The Oppia Team<br>'
            '<br>'
            'You can change your email preferences via the '
            '<a href="https://www.example.com">Preferences</a> page.'
        )
```